src/components/Alert/AlertMessage.tsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Alert } from "antd";
+
+export type AlertProps = {
+  message: string;
+  description: string;
+  type: "success" | "info" | "warning" | "error";
```

Comment:
типы можно вынести в enum AlertType {success='success', ...} и использовать
```suggestion
  type: AlertType
```

Reply:
Все остальные замечания исправил. Спасибо за code review !